test/blackbox-tests/test-cases/dune-project-meta/github11205.t

```diff
@@ -0,0 +1,64 @@
+Support for negation in package dependencies:
+
+  $ cat > dune-project <<EOF
+  > (lang dune 3.18)
+  > (generate_opam_files)
+  > (package
+  >   (name foo)
+  >   (allow_empty)
+  >   (depends
+  >    (ocp-indent
```

Comment:
Can you add a test when we have a package named "not" with a constraint? E.g. `(not (>= 3.0))`

Reply:
Done!